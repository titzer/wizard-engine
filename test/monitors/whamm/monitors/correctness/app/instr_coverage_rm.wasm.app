../../../coverage_monitor1.wasm
