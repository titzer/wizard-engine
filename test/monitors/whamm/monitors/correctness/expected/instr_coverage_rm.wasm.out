func #0:
  +3   reached
  +5   reached
  +7   reached
  +9   reached
  +11   reached
  +13   reached
  +14   reached
  +16   reached
  +18   reached
  +20   reached
  +21   reached
  +23   reached
  +25   reached
  +27   reached
  +28   reached
  +30   reached
  +32   reached
  +34   reached
  +36   reached
  +38   reached
func #1:
  +1   reached
  +3   reached
  +5   reached
  +6   reached
  +7   reached
  +8 unreached
  +9   reached
  +10   reached
func #2:
  +1   reached
  +3   reached
  +5   reached
  +6   reached
  +8   reached
  +10 unreached
  +11   reached
func #3:
  +1 unreached
func #4:
  +1 unreached
  +2 unreached
